exporter/awsprometheusremotewriteexporter/auth.go

```diff
@@ -104,8 +105,16 @@ func newSigningRoundTripper(cfg *Config, next http.RoundTripper, runtimeInfo str
 }
 
 func getCredsFromConfig(auth AuthConfig) (*credentials.Credentials, error) {
+	config := aws.Config{Region: aws.String(auth.Region)}
+	if auth.STSEndpoint != "" {
```

Comment:
I suppose there isn't a good way to write a unit test for this logic?

Reply:
The creation of the `aws.Config` could be extracted into a separate function and a test written to ensure that the `config.STSRegionalEndpoint` value is set appropriately.

Reply:
I've added a new unit test.